xnmt/output.py

```diff
@@ -60,3 +60,14 @@ def __init__(self, merge_indicator=u"@@"):
 
   def words_to_string(self, word_list):
     return u" ".join(word_list).replace(self.merge_indicator_with_space, u"")
+
+class JoinedPieceTextOutputProcessor(PlainTextOutputProcessor):
```

Comment:
I don't think we need a separate JoinedPieceTextOutputProcessor, as it's basically the same as JoinCharTextOutputProcessor.

Reply:
JoinCharTextOutput is slightly different, because it only replaces a single space token with space. For sentence piece, a word could have a space token attached to it at the beginning, so we need to do a replacement after merging

Reply:
Ahh, right. Thanks!